lib/execution_engine2/db/models/models.py

```diff
@@ -332,6 +339,53 @@ def __repr__(self):
         return self.to_json()
 
 
+# class BatchJobCollection(Document):
```

Comment:
Obv this is commented out so we don't need to deal with this now, but I don't really understand why a collection of containers is needed.

Also you may want to put these in a different collection so any future indexes aren't full of `null`s - or you aren't forced to use sparse indexes when you may not want to